hooks/post_gen_project.py

```diff
@@ -1,9 +1,149 @@
+import os
+import json
+import shutil
 import subprocess
 
+from python_terraform import Terraform
+
+YES_OPTIONS, NO_OPTIONS = frozenset(["y"]), frozenset(["n", ""])
+PERM_OPTIONS = frozenset(["push", "pull", "admin"])
+
+
+def multiple_choices(question):
+    selection = []
+    additional = "Would you like to add another (y/n)? [n]: "
+
+    multi_q_check = True
```

Comment:
we're already namespaced into the multliple_choices function, let's just call this `check`

```suggestion
    check = True
```